pickee_api/gateway.py

```diff
@@ -37,7 +37,11 @@ def get_available_providers(request):
 
         response = {}
         if data.get('results'):
-            relevant_result = data.get('results')[0]
+            result_index = 0
+            relevant_result = data.get('results')[result_index]
+            while relevant_result.get('name') != movie_name:
```

Comment:
Are comparing here the TMDB movie name with Utelly's movie name? It so maybe we should use some logic that verifies whether it is a close match.

Reply:
This is a good point. I'll look into it!